### PR TITLE
Added ability to see join date of other users

### PR DESCRIPTION
**Pull request by Fooooooooooooooo**

### CODE REVIEW

**Comment by DarkZek** [comment]
You should just be using 
IUser user = null
as the arguements and change the code depending on if the user is null or not. 0/10 foooo

**Comment by TheTrueColonel** [comment]
image

**Comment by Fooooooooooooooo** [comment]
all of you are BULLIES

File: DiscordBot/Modules/UserModule.cs [hunk 1]
```diff
@@ -209,19 +209,11 @@ private async Task DisplayProfile(IUser user)
...
-        {
-            var userId = Context.User.Id;
-            DateTime.TryParse(_databaseService.GetUserJoinDate(userId), out DateTime joinDate);
-            await ReplyAsync($"{Context.User.Mention} you joined **{joinDate:dddd dd/MM/yyy HH:mm:ss}**");
-            await Context.Message.DeleteAsync();
-        }
         
-        [Command("joindate"), Summary("Display join date of mentioned user. Syntax : !joindate @user")]
+        [Command("joindate"), Summary("Display your join date or join date of mentioned user. Syntax : !joindate @user")]
         private async Task JoinDate(IUser user)
```
Comment (TheTrueColonel) [inline DiscordBot/Modules/UserModule.cs]: `private async Task JoinDate(IUser user = null)`

You're bad

**Comment by Sirush** [comment]
It's intentional that this doesn't exist as stated many times before.

**Pull request closed without merging**